### DISCUSSION TITLE
Include Webp as a Built-in delegate with Linux distribution (AppImage)

### DISCUSSION THREAD

**Discussion by williamjacobszoon** [ImageMagick/ImageMagick#4096] [Help] [answered]
Hi All,

Apologies first of all, Not sure whether this is the correct place to post this request. This is a request to the developers to include the Webp as a built-in delegate for Linux distribution (AppImage) which can be downloaded from (https://download.imagemagick.org/ImageMagick/download/binaries/magick)

```
Version: ImageMagick 7.1.0-4 Q16 x86_64 2021-07-18 https://imagemagick.org
Copyright: (C) 1999-2021 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC HDRI OpenMP(4.5)
Delegates (built-in): bzlib djvu fontconfig freetype jbig jng jpeg lcms lqr lzma openexr png raqm tiff x xml zlib
```

**Comment by dlemstra** [comment]
If anyone would want to pick this up they can find the build for the AppImage here: https://github.com/ImageMagick/ImageMagick/blob/main/.github/workflows/app-image.yml

**Comment by williamjacobszoon** [comment, marked as answer]
Thanks Cristy for including the webp as a built-in delegate.

[commit dc1df68](https://github.com/ImageMagick/ImageMagick/commit/dc1df68764b537d90be3311f6413c4271ef1ad22)

`apt-get install -y autoconf curl fuse git kmod libbz2-dev libdjvulibre-dev libfontconfig-dev libfreetype6-dev libfribidi-dev libharfbuzz-dev liblcms-dev libopenexr-dev libturbojpeg0-dev liblqr-dev libraqm-dev libtiff-dev libwebp-dev libx11-dev libxml2-dev liblzma-dev software-properties-common wget`
